tests/trx2junit.Tests/Extensions/TimeExtensionsTests/ParseDateTime.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using NUnit.Framework;
+
+namespace trx2junit.Tests.Extensions.TimeExtensionsTests
+{
+    [TestFixture]
+    public class ParseDateTime
+    {
+        [Test]
+        public void Valid_JUnitDateTime___OK()
+        {
```

Comment:
Add also a brute-fore test like in https://github.com/gfoidl/trx2junit/blob/f43fc7543998f1f101f4cbd803ac96cc787eed1f/tests/trx2junit.Tests/Extensions/TimeExtensionsTests/ToJUnitDateTime.cs#L23